config/initializers/carrierwave.rb

```diff
@@ -8,20 +8,20 @@
   config.enable_processing = !Rails.env.test?
 end
 
-# Setup CarrierWave to use Amazon S3. Add `gem "fog-aws" to your Gemfile.
-#
-# CarrierWave.configure do |config|
-#   config.storage = :fog
-#   config.fog_provider = 'fog/aws'                                             # required
-#   config.fog_credentials = {
-#     provider:              'AWS',                                             # required
-#     aws_access_key_id:     Rails.configuration.secrets.aws_access_key_id,     # required
-#     aws_secret_access_key: Rails.configuration.secrets.aws_secret_access_key, # required
-#     region:                'eu-west-1',                                       # optional, defaults to 'us-east-1'
-#     host:                  's3.example.com',                                  # optional, defaults to nil
-#     endpoint:              'https://s3.example.com:8080'                      # optional, defaults to nil
-#   }
-#   config.fog_directory  = 'name_of_directory'                                 # required
-#   config.fog_public     = false                                               # optional, defaults to true
-#   config.fog_attributes = { 'Cache-Control' => "max-age=#{365.day.to_i}" }    # optional, defaults to {}
-# end
+if Rails.env.production?
+  require 'carrierwave/storage/fog'
+
+  CarrierWave.configure do |config|
```

Comment:
I would copy this from https://github.com/HospitaletDeLlobregat/decidim-hospitalet/blob/master/config/initializers/carrierwave.rb as the newest carrierwave/fog/whatever versions gave me quite a hard time.